docs/spec/spec.md

```diff
@@ -0,0 +1,309 @@
+# Object Model
+
+This is the detailed specification of the Knative Eventing Object Model. For
+context, see the [overview](overview.md), [motivation](motivation.md), and the
+[interface contract](interfaces.md) documents.
+
+ * [Source](#kind-source) 
+ * [Channel](#kind-channel)
+ * [Subscription](#kind-subscription)
+ * [Provider](#kind-provisioner)
+
+---
+
+## kind: Source
+
+### group: eventing.knative.dev/v1alpha1
+
+_Describes a specific configuration (credentials, etc) of a source system which
+can be used to supply events. Sources emit events using a channel specified in
```

Comment:
Rather than "using a channel specified" I'd say:

```markdown
A common pattern is for Sources to emit events to Channel to allow event
delivery to be fanned-out within the cluster.
```